Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveSwift
+import Result
+
+public protocol PaymentMethodsViewModelInputs {
+  func viewDidLoad()
+  func didTapAddNewCardButton()
+}
+
+public protocol PaymentMethodsViewModelOutputs {
+  /// Emits the user's stored cards
+  var didFetchPaymentMethods: Signal<[GraphUserCreditCard.CreditCard], NoError> { get }
```

Comment:
This output should just be called `paymentMethods`, I think from an interface perspective we aren't concerned with whether they were fetched, just that this emits values.